playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -30,6 +33,7 @@
     command: oc scale --replicas=0 dc/docker-registry
 
   - name: Create S3 bucket
+    when: aws_create_bucket | bool
```

Comment:
I'm not sure I understand the purpose of this 'when' statement. Without it, the bucket will only be created if it does not already exist. So it serves as a verification that the registry's backend storage actually exists. Is there a case where we would specifically *not* want to verify that the bucket exists?